pages/index.js

```diff
@@ -1,30 +1,86 @@
-import React from "react";
+import React, { useState } from "react";
 
-import TodoInput from "../components/todo-input";
+import MemoizedTodoInput from "../components/todo-input";
 import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
 
 export default function Home() {
+  const [date, setDate] = useState("");
+  const [todo, setTodo] = useState("");
+  const [todoID, setTodoID] = useState(0);
+  const [todoList, setTodoList] = useState([]);
+
+  // 값이 바뀔때마다 실행
+  const onChangeTodo = (e) => {
+    setTodo(e.target.value);
+  };
+  const onChangeDate = (e) => {
+    //입력을 8자리 넘어가면 받지 않도록 설정
+    if (String(e.target.value).length <= 8) setDate(e.target.value);
+  };
```

Comment:
1. 입력된 값이 바뀔때마다 발동되는 함수가, page component에 선언돼야할까요?
이 로직은 input component에서만 갖고있어도 될 것 같아요!

2. 위 두개 함수를 1개로 합칠 수 있습니다! name property를 활용해주시면 되세요 ㅎ
```suggestion
  const handleFormChange = (e) => {
    setForm({
      [e.target.name] : e.target.value
    });
  };
```
자세한 사용 방법은 오늘 저녁에 업로드될 모범답안 참고해주세요~!